### ISSUE TITLE
Can't list registries in structured output

### ISSUE THREAD

**Issue by jeffmaury** [redhat-developer/odo#5993]
/kind bug

odo integrations (OpenShift Connector) used `odo registry list -o json` with odo v2.x

## What versions of software are you using?

**Operating System:**
Win10Pro

**Output of `odo version`:**

```
odo v3.0.0-beta2 (c8e4458f4)

Server: https://api.sandbox-m2.ll9k.p1.openshiftapps.com:6443
Kubernetes: v1.23.5+9ce5071
```

## How did you run odo exactly?

`odo preference view -o json`

## Actual behavior

```
C:\work\src\tmp\odotest\demo>odo preference view -o json
 X  Machine readable output is not yet implemented for this command
```

## Expected behavior

Structured output should be seen

**Comment by rm3l** [comment]
/remove-kind bug
/kind feature

JSON output for `odo preference view` needs to be implemented.

**Comment by dgolovin** [comment]
@kadel we cannot migrate to odo v3 without support `-o json` output, unless we going to remove `Registries View` which shows list of the available registries and let user modify it (add/remove/edit entries).

What happened to `odo registry list -o json`? It was working fine in v2.

**Comment by rm3l** [comment]
We will be working on this issue during this sprint. `odo preference view -o json` should be available for v3.0.0-rc2.
